src/info/info.ts

```diff
@@ -63,15 +63,24 @@ export class EnvironmentInfo {
 
     /**
      * Create a new [[EnvironmentInfo]] inside the DOM element with given `id`
-     * @param id         HTML id of the DOM element where the sliders and
+     * @param element    HTML id of the DOM element where the sliders and
```

Comment:
the documentation needs to be updated to say something like `HTML element or string 'id' of such element ...`